anonymizer/base.py

```diff
@@ -262,6 +262,18 @@ def get_queryset(self):
             qs = qs.order_by('id')
         return qs
 
+    def get_queryset_iterator(self, chunksize=1000):
+        queryset = self.get_queryset().order_by('pk')
+        last = queryset.last()
+        if last is None:
+            return
+
+        pk = 0
+        while pk < last.pk:
+            for row in queryset.filter(pk__gt=pk)[:chunksize]:
```

Comment:
This function looks like it works, but something about it reads oddly. How about something like 

```python
queryset = ...
num_rows = queryset.count()
index = 0

while index < num_rows:
    rows = queryset[index:index + chunksize]
    index += chunksize    
    for row in rows:
        yield row
```